packages/controller/src/resources/monitoring/system/dashboards/cortex-overview-per-tenant.json

```diff
@@ -158,7 +158,7 @@
       "reverseYBuckets": false,
       "targets": [
         {
-          "expr": "sum by (le) (rate(cortex_api_proxy_request_duration_seconds_bucket{job=\"$tenant-tenant.cortex-api\",method=\"GET\",status_code=\"200\",route=~\"api_prom_api_v1_query\"}[$timewindow]))",
+          "expr": "sum by (le) (rate(cortex_api_proxy_request_duration_seconds_bucket{job=\"$tenant-tenant.cortex-api\",method=\"POST\",status_code=\"200\",route=~\"api_v1_push\"}[$timewindow]))",
```

Comment:
Why this change?

Reply:
The center column in these dashboards is supposed to show the read direction, as indicated in the title of this panel:

```
API proxy /api/prom/api/v1/query, 200-acked GETs, request duration, 
```
changing this to keep track of POST requests would result in a pretty dangerous mismatch between title and actual plot :)

Reply:
This was a mistake, reverting.